lib/stripe/stripe_object.rb

```diff
@@ -72,13 +72,17 @@ def refresh_from(values, opts, partial=false)
     #
     # * +values+ - Hash of values to use to update the current attributes of
     #   the object.
+    # * +:opts+ Options for StripeObject like an API key.
     #
     # ==== Options
     #
-    # * +:opts+ Options for StripeObject like an API key.
-    def update_attributes(values, opts = {})
+    # * +:dirty+ Whether values should be initiated as "dirty" (unsaved).
+    #   Defaults to true.
+    def update_attributes(values, opts = nil, options = {})
```

Comment:
I changed it back to `opts = {}`. I think I just ended up refactoring this so many times that I didn't really notice that needless change.

I've also renamed `options` to `method_options`. This is unfortunately a hard one because we use the `opts` convention all over the place, and the Ruby idiom for method options is very much `options` as well. I think this will get more clear once I shave the yak to get Ruby 1.9 support pulled out of here because we can then move this over to a named optional argument.